src/components/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -1,24 +1,24 @@
 
 <template>
   <div
-    class="segmented-control d-flex"
-    :class="{ 'allow-pointer-events': allowPointerEvents}"
+    class="k-segmented-control segmented-control d-flex"
```

Comment:
@kaiarrowood out of curiosity, why do we need both `k-segmented-control` and `segmented-control` classes here?

Reply:
@kaiarrowood this is a good question since this doesn't follow the `.k-*` syntax. I know it's out of scope, but would you mind updating all instances of `.segmented-control` with `.k-segmented-control`? (I think it's just here and in the tests)

Reply:
Oh, good call 🤔
I don't have any objections if we leave that class there, since it doesn't hurt us. Thoughts @adamdehaven?

Reply:
@kaiarrowood let's delete it; all of our classes are documented as `k-*` anyway

Reply:
Done and ready for final review 🚀